### PR TITLE
Use old-releases.ubuntu.com in Ubuntu containers

### PR DESCRIPTION
**Pull request by friedc**
Releases which are end of life can only be accessed through old-releases.ubuntu.com. See:

* https://ubuntu.com/about/release-cycle
* https://help.ubuntu.com/community/EOLUpgrades.

### CODE REVIEW

File: docker/ubuntu-12.04/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
+RUN sed -i -re 's/([a-z]{2}\.)?archive.ubuntu.com|security.ubuntu.com/old-releases.ubuntu.com/g' /etc/apt/sources.list
+
 RUN apt-get update && apt-get install -y \
   git \
   bzip2 \
   build-essential \
   m4
 
-RUN git clone https://github.com/fbenz/FpDebug.git
```
Comment (fbenz) [inline docker/ubuntu-12.04/Dockerfile]: Please revert this change. It makes sense on your fork but not when merging back.

**Review by fbenz — COMMENTED** [review]
Thanks for fixing the build. The `sed` expression is matching the following answer https://askubuntu.com/a/91821 and makes sense to me. Please see my comment.

File: docker/ubuntu-12.04/Dockerfile (hunk 1 above)
Comment (friedc) [reply docker/ubuntu-12.04/Dockerfile]: Done. I should have used a separate branch.

**Review by fbenz — APPROVED** [review]
